fares-data-build-tool/fdbt-site
# feat: added signposts to BODS

# Description

-   This PR adds signposting to BODS on the site. These signposts are added to the `/home`, `/contact` and `/thankyou` pages.

# Testing instructions

-   Pull down this branch and run the site locally. Check that each of the required signposts have been added on the respective pages of the site.

# Type of change


# Checklist:
